### ISSUE TITLE
Create accessibility checklist

### ISSUE THREAD

**Issue by gedankenstuecke** [alan-turing-institute/AutSPACEs#369]
### Summary

Once we have a working MVP, we need to check that it fulfills the accessibility guidelines that have emerged both from general resources, but also the past user testing and focus groups. Beyond testing the MVP once done, this checklist should also be used to evaluate changes to the platform.

### Deadline 

Should be worked on once MVP is close to being ready. 

### Resources

Existing Guidelines can be found in a variety of places and will need to be merged and prioritized. Amongst others this includes the [gov.uk accessibility checklist](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/accessibility/gov-checklist.md) and there is also a wide range of potential topics spread across the [accessibility issue label](https://github.com/alan-turing-institute/AutSPACEs/labels/accessibility) that will need to be synthesized. 

Topics include: 
* Consistent form elements (#53) 
* text descriptions for images/audio (#49, #48, #47)
* page titles (#46)
* Support for text-to-speech screenreaders (#325 #308)
